src/progressBar.js

```diff
@@ -18,7 +18,7 @@ class ProgressBarController {
     }
 
     // Don't do any of this while testing
-    if (process.env.NODE_ENV === "test") {
+    if (process.env.NODE_ENV === "lerna-test") {
```

Comment:
Filed https://github.com/lerna/lerna/issues/447.  Thanks for looking at this @taion.